models/recurly__account_overview.sql

```diff
@@ -10,34 +10,34 @@ account_history as (
     from {{ var('account_history') }}
 ),
 
-transactions_grouped as (
+transactions_grouped as ( 
```

Comment:
@fivetran-joemarkiewicz  Can do. Is there a best practice for when a model should be declared intermediate? Like X number of lines?

Reply:
Great question, but we don't necessarily have a "X means intermediate model". Our rule of thumb is mainly that we don't want to do too much in a single model if it can be avoided. My methodology typically is the following:
- Can this logic be repurposed for other downstream models? If so, let's make it an intermediate model.
- Is this model doing a lot and can feasibly be broken down into it's own components via intermediate models? Of so, let's make it an intermediate model.

Mainly, making models intermediate (ephemeral) should be done to make our code dry and well organized.

Reply:
@fivetran-joemarkiewicz Got it, so `monthly_recurring_revenue` for example does have multiple CTEs in it for example, but since each CTE is only conducting one major operation instead of multiple and the use case is so specific, it makes sense not to have a CTE?

Reply:
Yeah exactly! Those cte's are very specific and make sense in my opinion to exist in the end model.